corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -40,10 +40,11 @@ var PricingTable = function (options) {
     self.oShowAnnualPricing = ko.observable(options.currentIsAnnualPlan);
 
     self.oIsSubmitDisabled = ko.computed(function () {
+        var isSubscribablePlan = !!self.oSelectedEdition() && !['free', 'enterprise'].includes(self.oSelectedEdition());
```

Comment:
very small nit, but if we're introducing new variables, I think we should try to use modern `let` or `const` rather than `var`

Reply:
I agree, but I was more interested in consistency with the surrounding code here. In an ideal world I'd find the time to update this whole file to use `let` and `const` as appropriate.